gaia-fast-list.js

```diff
@@ -865,6 +917,63 @@ Internal.prototype = {
     img.style.opacity = 0;
   },
 
+  normalizeImageSrc(src) {
+    if (typeof src == 'string') return src;
+    else if (src instanceof Blob) return URL.createObjectURL(src);
+    else throw new Error('image src invalid');
+  },
+
+  getCachedImage(index) {
+    debug('get cached image', index);
+    return this.images.hash[index];
+  },
+
+  cacheImage(index, raw) {
+    debug('cache image', index, raw);
```

Comment:
should we add an early return if the `imageCacheSize` is 0?
(if somebody want's to disable the cache)